addons/BlocklyCodeEditor/src/presenter.js

```diff
@@ -171,11 +172,10 @@ function AddonBlocklyCodeEditor_create () {
 
         isPreviewDecorator(presenter.setConfiguration)(presenter.configuration.initialConfiguration);
 
-        presenter.view.addEventListener('DOMNodeRemoved', function onDOMNodeRemoved_Blockly (ev) {
-            if (ev.target === this) {
-                presenter.destroy();
-            }
-        });
+        if (!isPreview) {
+            MutationObserverSingleton.createObserver(presenter.destroy, this);
```

Comment:
Tutaj ten argument this nic nie robi, bo presenter.destroy nie przyjmuje żadnych paramentrów.